content/en/docs/reference/glossary/resource-claim-parameters.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Resource Claim Parameters
+id: resource-claim-parameters
```

Comment:
I'd use `ResourceClaimParameters` as the `id` as well.

Reply:
It would be inconsistent with the rest of glossary items.

Reply:
I'm saying, as an approver and tech lead for SIG Docs, that what I'm recommending is our ideal practice. We welcome help to bring other glossary items in line with our style guide, but we are not prioritizing that work.

Reply:
Fixed.

Reply:
This change might be causing the [build failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65ca821525786f0008b245f4). After looking at the [{{< glossary_definition >}}](https://github.com/kubernetes/website/blob/c105f1a03a7acddbc369c0834bccf624782a0fc6/layouts/shortcodes/glossary_definition.html) (applicable to _glossary_tooltip_ as well) shortcode file, it appears that the shortcode relies on matching the `term_id` passed with glossary entry **_[filenames](https://github.com/kubernetes/website/blob/c105f1a03a7acddbc369c0834bccf624782a0fc6/layouts/shortcodes/glossary_definition.html#L6)_** to retrieve the description.

With this recent change, the `id` in the file now reads as **"ResourceClaimParameters"** while the file name is **"resource-claim-parameters.md"**. Consequently, the shortcode is unable to locate the file to display the necessary data for `term_id="ResourceClaimParameters"`, leading to the build failure. To address this, I suggest temporarily reverting to the existing conventions. However, in the long run, we could potentially enhance the shortcode logic to match the `term_id` with the `id` within the glossary file content instead of relying solely on filename.

Reply:
Instead of reverting the id field, I can just rename the files to match the <id>.md, right ? I'll give it a shot.